### PR TITLE
Allow more verbose

### PR DESCRIPTION
**Pull request by jpereira**
perl -p -i -e 's/\t\@/\t\${Q}/' $(find . -name "all.mk" -print)

### CODE REVIEW

**Comment by arr2036** [comment]
The majority of the commands you removed the @ from were echo or printf, which is wrong.  We don't want:

```
echo DOXYGEN /path/to/doxygen
DOXYGEN /path/to/doxygen
```

in the output...

File: src/lib/all.mk [hunk 1]
```diff
@@ -19,7 +19,7 @@ src/freeradius-devel: | src/include/${1}
 
 src/include/${1}:
 	$${Q}[ -e $$@ ] || ln -sf $${top_srcdir}/src/lib/${1} $$@
-	@echo LN-SF src/lib/${1} $$@
+	${Q}echo LN-SF src/lib/${1} $$@
```
Comment (arr2036) [inline src/lib/all.mk]: These all need to be escaped $${Q}

Comment (arr2036) [reply src/lib/all.mk]: But still, you're emitting the echo command... so not necessary to begin with

**Comment by jpereira** [comment]
Got it @arr2036 . I updated and fixed some extra things like: `${Q}${Q}command....` and now we are using `@echo ...` only to print the first target message.

**Pull request merged**